plugins/catalog-backend/src/service/router.ts

```diff
@@ -107,6 +122,8 @@ export async function createRouter(
         res.status(200).json(entities[0]);
       })
       .delete('/entities/by-uid/:uid', async (req, res) => {
```

Comment:
This one is a bit tricky, as it will be needed to delete entities even in readonly mode. That's because this is currently and will likely going forward also be the main way to delete entities that have been removed from YAML files or other declarations.

Reply:
Interesting. I've given this some though since creating this PR and had a hard time figuring out what to do with the button.

As discussed in #3348 we would expect the processor to remove entities no longer tracked in configuration files at some point to allow for a true declarative model. But leaving this in for now is viable I guess. 

It is not possible to remove entities that was added through the `locations` configuration right now, right? That would pose a problem I think.

Reply:
Should I allow this operation for now and we can come back to it later on? Maybe with a TODO or something?

Reply:
Yep leave it in for now.